### ISSUE TITLE
Base64 generating hundreds of image attachments not used in email message body

### ISSUE THREAD

**Issue by TechBill** [mautic/mautic#11381]
### Mautic Version

4.4.x series

### PHP version

7.4.+

### What browsers are you seeing the problem on?

Not relevant

### What happened?

After switching to Base64, we were informed by several of our subscribers that they are receiving our newsletter with hundreds of attachments when we only have 7 or 8 images in the message body. 

One header image, small logo, and several social icons is all that in the message body. 

By looking in the email source we can see that there are over 140 attachment images that are not in the message body but it all seem to be a small image. 

I have attached a copy/paste from the email source just the bottom part of the email source with all those image attachment for your reviews and none of those images were added in by us or exist on message body but it showing up as an attachment for numbers of our subscribers (seeing 140 paperclip icons across the screen etc.)

[email-source.txt](https://github.com/mautic/mautic/files/9263371/email-source.txt)

### How can we reproduce this issue?

Step 1: Enable `Convert image to Base64` under Configuration - Email Setting
Step 2: Send out email with a couple of image or social gif icons etc to yourself
Step 3: Review email source to see if there exist a numerous of invisible image attachments. 

### Relevant log output

```shell
Nothing shown in log seems to be related to this issue.
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by TechBill** [comment]
Due to its' small size, I am wondering if that a bugged out tracking pixel being repeated 

and tracking pixel cannot be Base64 correct?

**Comment by TechBill** [comment]
I just noticed something else so I wanted to add it here.

We average about 55% to 60% of our newsletter read which is correct.

When we enabled the Base64, our newsletter read would jump up to 99% to 100% .  It obviously something to do with repeated tracking pixel being bugged out and causing the statistics to be off. 

See attached screenshot 

Without Base64 -past few weeks 
![Screen Shot 2022-08-04 at 4 42 07 PM](https://user-images.githubusercontent.com/945103/182958027-7346a69c-673f-4159-8934-53c4a7a26a43.jpg)


With Base64 enabled - this week
![Screen Shot 2022-08-04 at 4 42 25 PM](https://user-images.githubusercontent.com/945103/182958087-b20221d7-ad46-486d-ba02-63ec5febf193.jpg)
